nautobot/core/testing/views.py

```diff
@@ -1321,12 +1320,11 @@ def test_bulk_delete_form_contains_all_filtered(self):
             # Expect a 200 status cause we are only rendering the bulk delete table after pressing Delete Selected button.
             self.assertHttpStatus(response, 200)
             response_body = utils.extract_page_body(response.content.decode(response.charset))
-            # Check if the first and second pk is passed into the form.
-            self.assertIn(f'<input type="hidden" name="pk" value="{first_pk}"', response_body)
-            self.assertIn(f'<input type="hidden" name="pk" value="{second_pk}"', response_body)
-            self.assertIn("<strong>Warning:</strong> The following operation will delete 2 ", response_body)
-            # Check if the third pk is not passed into the form.
+            # Check if all pks is not part of the html.
+            self.assertNotIn(f'<input type="hidden" name="pk" value="{first_pk}"', response_body)
+            self.assertNotIn(f'<input type="hidden" name="pk" value="{second_pk}"', response_body)
             self.assertNotIn(f'<input type="hidden" name="pk" value="{third_pk}"', response_body)
+            self.assertIn("<strong>Warning:</strong> The following operation will delete 2 ", response_body)
```

Comment:
`self.assertInHTML('<input type="hidden" name="_all" value="true" />', response_body)` could be added to this test as well.